### PR TITLE
[safebreach-129] SafeBreach fix small bugs

### PR DESCRIPTION
**Pull request by gtsafeB**
## XSOAR [Partner](https://xsoar.pan.dev/docs/partners/become-a-tech-partner)?
In case you are a technological partner:
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**.

If you are not a technological partner (i.e. PANW employee, customer or individual developer), please delete this section.

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: fix calculation of the total simulations number and 'indicatorLimit' params was cast to default value(1000) if not provided.

## Minimum version of Demisto
- [ ] 5.0.0
- [x] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by barchen1 — CHANGES_REQUESTED** [review]
Thanks for your contribution!

File: Packs/SafeBreach/Integrations/SafeBreach_v2/SafeBreach_v2.py [hunk 1]
```diff
@@ -331,7 +331,11 @@ def get_indicators_command(client: Client, insight_category: list, insight_data_
             Returns:
                 List[Dict] -- List of insights from SafeBreach
             """
-    limit: int = int(args.get('limit') or demisto.params().get('indicatorLimit'))
+    indicator_limit_param = demisto.params().get('indicatorLimit')
+    if indicator_limit_param == '':
+        indicator_limit_param = 1000
```
[the remark this answers is not in the document]
Comment (gtsafeB) [reply Packs/SafeBreach/Integrations/SafeBreach_v2/SafeBreach_v2.py:336]: I'm not sure that's correct when indicatorLimit is not provided in the configure screen its set as '' (=empty string) which get() will return a value and the default value will not be returned.

[the remark this answers is not in the document]
Comment (barchen1) [reply Packs/SafeBreach/Integrations/SafeBreach_v2/SafeBreach_v2.py:336]: So you can edit the indicatorLimit param and add an initial value.

Screen Shot 2020-09-30 at 11 35 16


Cause 1000 can cause to an overload, 200 is our default.

**Review by barchen1 — CHANGES_REQUESTED** [review]

File: Packs/SafeBreach/Integrations/SafeBreach_v2/SafeBreach_v2.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (gtsafeB) [reply Packs/SafeBreach/Integrations/SafeBreach_v2/SafeBreach_v2.py:336]: Great, let's do it that way.
I'll keep the initial value to be 1000 if that's ok

[the remark this answers is not in the document]
Comment (barchen1) [reply Packs/SafeBreach/Integrations/SafeBreach_v2/SafeBreach_v2.py:336]: np, you can keep it as 1000

**Review by barchen1 — CHANGES_REQUESTED** [review]

**Review by barchen1 — APPROVED** [review]

**Pull request merged**